src/app/containers/NewStoryPromo/PromoOrganisms/TopStoriesPromo/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { arrayOf, shape } from 'prop-types';
+import { storyItem } from '#models/propTypes/storyItem';
+import pathOr from 'ramda/src/pathOr';
+import { C_GREY_2 } from '#app/legacy/psammead-styles/src/colours';
+import Promo from '../../Promo';
+import { TopStoriesPromoWrapper } from './index.styles';
+import TopStoriesItem from './TopStoriesMolecules/TopStoriesItem';
+import TopStoriesList from './TopStoriesMolecules/TopStoriesList';
+
+const TopStoriesPromo = ({ content }) => {
+  const { script, service, dir, translations } = useContext(ServiceContext);
+  const title = pathOr('Top Stories', ['topStoriesTitle'], translations);
+  const hasSingleContent = content.length === 1;
+  const LABEL_ID = 'top-stories-heading';
+
+  return (
+    <Promo Wrapper={TopStoriesPromoWrapper}>
+      <Promo.SectionLabel
+        script={script}
+        service={service}
+        dir={dir}
+        labelId={LABEL_ID}
+        columnType="secondary"
+        backgroundColor={C_GREY_2}
```

Comment:
Making the component stylable in emotion is more flexible, and will reduce the complexity / prop count needed within the component:

```
const StyledSectionLabel = styled(Promo.SectionLabel)`
  background: ${C_GREY_2}
`
```

Reply:
Since the line that was causing the need for this weird styling convention was removed, I believe this Property can be removed. I will create a new ticket to deal with this tomorrow.